src/mappers/culture/culture_graphics_mapping.h

```diff
@@ -0,0 +1,61 @@
+#ifndef SRC_MAPPERS_CULTURE_CULTUREGRAPHICSMAPPING_H
+#define SRC_MAPPERS_CULTURE_CULTUREGRAPHICSMAPPING_H
+
+
+
+#include <string>
+
+
+namespace mappers
+{
+struct IdeologyPortraitPaths
+{
+   std::set<std::string> communism;
```

Comment:
vector can be useful here, as it allows us to more easily select different portraits for different characters and countries.